azure/cloud/dbsecurityalertpolicies.rego

```diff
@@ -128,185 +118,153 @@ package rule
+    "Resource Type": "microsoft.sql/servers/databases/securityalertpolicies",
+    "Policy Help URL": "",
+    "Resource Help URL": "https://docs.microsoft.com/en-us/azure/templates/microsoft.sql/2018-06-01-preview/servers/databases/securityalertpolicies"
+}
 
 #
 # PR-AZR-CLD-SQL-021
-# Not valid for cloud provider as cloud seperates all the child resources into seperate resource
-
-# default sql_alert = null
-
-# azure_issue["sql_alert"] {
-#     resource := input.resources[_]
-#     lower(resource.type) == "microsoft.sql/servers/databases"
-#     sql_resources := resource.resources[_]
-#     lower(sql_resources.type) == "securityalertpolicies"
-#     not sql_resources.properties.emailAccountAdmins
-# }
-
-
-# azure_issue["sql_alert"] {
-#     resource := input.resources[_]
-#     lower(resource.type) == "microsoft.sql/servers/databases"
-#     sql_resources := resource.resources[_]
-#     lower(sql_resources.type) == "securityalertpolicies"
-#     sql_resources.properties.emailAccountAdmins != true
-# }
-
-
-# sql_alert {
-#     resource := input.resources[_]
-#     lower(resource.type) == "microsoft.sql/servers/databases"
-#     sql_resources := resource.resources[_]
-#     lower(sql_resources.type) == "securityalertpolicies"
-#     not azure_attribute_absence["sql_alert"]
-#     not azure_issue["sql_alert"]
-# }
-
-
-# sql_alert = false {
-#     azure_attribute_absence["sql_alert"]
-# }
-
-
-# sql_alert = false {
-#     azure_issue["sql_alert"]
-# }
 
-# sql_alert_err = "microsoft.sql/servers/databases/securityalertpolicies property 'emailAccountAdmins' need to be exist. Its missing from the resource." {
-#     azure_attribute_absence["sql_alert"]
-# } else = "Threat Detection alert currently is not configured to sent notification to the sql server account administrators" {
-#     azure_issue["sql_alert"]
-# }
-
-# sql_alert_metadata := {
-#     "Policy Code": "PR-AZR-CLD-SQL-021",
-#     "Type": "Cloud",
-#     "Product": "AZR",
-#     "Language": "",
-#     "Policy Title": "Threat Detection alert should be configured to send notifications to the SQL server account administrators",
-#     "Policy Description": "Ensure that threat detection alert is configured to send notifications to the sql server account administrators",
-#     "Resource Type": "microsoft.sql/servers/databases/securityalertpolicies",
-#     "Policy Help URL": "",
-#     "Resource Help URL": "https://docs.microsoft.com/en-us/azure/templates/microsoft.sql/2018-06-01-preview/servers/databases/securityalertpolicies"
-# }
+default sql_alert = null
+
+azure_issue["sql_alert"] {
```

Comment:
@rezoan please change ```azure_issue``` to ```azure_attribute_absence```
also please do it for IaC as I checked

Reply:
All other changes were fine as I checked

Reply:
@shahinshirvani done.